javascript-data-model/order-history.js

```diff
@@ -0,0 +1,97 @@
+var orderHistory = [
+  [
+    {
+      orderNumber: 11428755,
+      total: 44.53,
+      orderPlacedDate: 'July 19, 2020',
+      deliveryDate: 'Delivered Jul 20 2020',
+      shipTo: 'JS Masher',
+      deliveredStatus: true,
+      returnWindow: 'Aug 19',
+      buyAgain: '',
+      orderDetails: '',
+      item: [
+        {
+          type: 'book',
+          price: 41.33,
+          invoice: '',
+          writeProductReview: '',
+          returnWindow: 'Sept7, 2020',
+          additionalInfo: {
+            author: 'Alexander, Christopher',
+            title: 'The timeless Way of Building',
+            productReview: '',
+            viewItem: ''
+          }
+        }
+      ]
+    }
+  ],
```

Comment:
Hmmm, this is the item for the second order, it looks like you are missing your first order's object 🤔